editor/editor_help.cpp

```diff
@@ -2143,16 +2281,49 @@ static void _add_text_to_rt(const String &p_bbcode, RichTextLabel *p_rt, Control
 
 			pos = brk_end + 1;
 			tag_stack.push_front("url");
-		} else if (tag == "img") {
+		} else if (tag.begins_with("img")) {
+			int width = 0;
+			int height = 0;
+			if (tag.length() > 4) {
+				Vector<String> subtags = tag.substr(4, tag.length()).split(",");
+				if (subtags.size() >= 1) {
+					width = subtags[0].to_int();
+				}
+				if (subtags.size() >= 2) {
+					height = subtags[1].to_int();
+				}
+			}
 			int end = bbcode.find("[", brk_end);
 			if (end == -1) {
 				end = bbcode.length();
 			}
 			String image = bbcode.substr(brk_end + 1, end - brk_end - 1);
 
-			Ref<Texture2D> texture = ResourceLoader::load(base_path.path_join(image), "Texture2D");
-			if (texture.is_valid()) {
-				p_rt->add_image(texture);
+			if (image.begins_with("http://") || image.begins_with("https://")) {
+				Ref<Texture2D> texture;
+				String image_cache_name = EditorPaths::get_singleton()->get_cache_dir().path_join(image.uri_encode());
+
+				// Try loading from cache.
```

Comment:
What if the remote image was updated? There should be a way to detect this or at least invalidate the cache (although invalidation can be achieved by adding `?whatever` to the URL; doing it manually is not ideal tho).